.gitignore

```diff
@@ -0,0 +1,3 @@
+test.md
+sth.md
+node_modules/
```

Comment:
Great Job here 👏 !!! You did well to add `test.md` and `node_modules` inside `.gitignore` file. but you pushed the `test.md file` and this is not part of the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md)

- [ ] Please make sure that `test.md` is not present in your GitHub repository.

Reply:
Thanks for your comments. The test.md file appears on the .gitignore file. This is ok, right?